content/en/news/releases/1.12.x/announcing-1.12.2/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Announcing Istio 1.12.2
+linktitle: 1.12.2
+subtitle: Patch Release
+description: Istio 1.12.2 patch release.
+publishdate: 2022-01-10
+release: 1.12.2
+aliases:
+    - /news/announcing-1.12.2
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.12.1 and Istio 1.12.2
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** privileged flag to Istio-CNI Helm charts to set `securityContext` flag.
+  ([Issue #34211](https://github.com/istio/istio/issues/34211))
+
+- **Fixed** an issue where enabling tracing with telemetry API will cause malformed host header being used at the trace report request.
```

Comment:
```suggestion
- **Fixed** an issue where enabling tracing with telemetry API would cause a malformed host header being used at the trace report request.
```